src/Extensions/AzureBlobStorage/AzureBlobStorage.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageTags>Diagnostic</PackageTags>
     <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
     <ToolCommandName>dotnet-monitor-egress-azureblobstorage</ToolCommandName>
+    <AssemblyName>dotnet-monitor-egress-azureblobstorage</AssemblyName>
```

Comment:
I inadvertently didn't update this when updating the `ToolCommandName` in the last set of changes.